receiver/jaegerreceiver/testdata/config.yaml

```diff
@@ -1,16 +1,37 @@
 receivers:
+  # The following demonstrates initializing the default jaeger receiver.
+  # By default all three protocols (grpc, thrift-http, thrift-tchannel)
+  # are enabled and the default endpoints are specified in factory.go
   jaeger:
+  # The following demonstrates specifying different ports.
+  # The Jaeger receiver only connects to ports and ignores addresses.
+  # However, an address is needed to specify for a proper configuration.
```

Comment:
":\<port\>" should work too. Please, make clear that this receiver will open the port on all available network interfaces, 127.0.0.1 in this sense is very misleading. The syntax of endpoint is "host:port" but host can be omitted.

Reply:
your wording is nicer - i'll use that to update this comment.